plone/app/event/dx/behaviors.py

```diff
@@ -44,10 +42,10 @@
 
 if PLONE5:
     # Plone 5
-    from plone.app.z3cform.widget import DatetimeWidget
+    from plone.app.z3cform.widget import DatetimeFieldWidget
 else:
     # Plone 4
-    from plone.app.widgets.dx import DatetimeWidget
+    from plone.app.widgets.dx import DatetimeFieldWidget
```

Comment:
True - But removing Plone 4 support is more than that. I'd like to make a separate PR for that.